src/features/shared-link-settings-modal/__tests__/ExpirationSection.test.js

```diff
@@ -95,5 +95,14 @@ describe('features/shared-link-settings-modal/ExpirationSection', () => {
 
             expect(wrapper.prop('inputProps')['data-prop']).toEqual('input');
         });
+
+        test('should pass dateFormat to DatePicker', () => {
```

Comment:
Is there any way to test this (and the tests in `SharedLinkSettingsModal`) via the UI instead of making assertions about internals? IMO, it's okay and even ideal for, e.g., Sonarqube to complain about ostensible gaps in coverage in order for us to write non-brittle tests that solely define user-facing functionality.